packages/plugin-protocol-http/src/utils.ts

```diff
@@ -59,3 +59,7 @@ export const abort = (request: MakeRequestResult) => {
 
     return meta && meta.requestAbort();
 };
+
+export const getCacheFlags = (request: MakeRequestResult): Record<string, any> => {
```

Comment:
nitpick: unused method

Reply:
It serves to indicate that the value comes from the cache, so we need it

Reply:
Sorry, can you please send link where `getCacheFlags` is used? Try to search multiple times(

Reply:
This is for future use, it will be useful to understand where we get the value from